### ISSUE TITLE
Update messes up screen

### ISSUE THREAD

**Issue by abailey-et** [mediachicken/sublimetext-defaultplus-theme#13]
With the latest update all the tabs overlap and when you move a tab one long streak is displayed. My sidebar disappears. I have to disable the theme for now so I can continue working.

I even commented out classic tabs and modern tabs settings, same deal.

![theme_errors](https://cloud.githubusercontent.com/assets/910352/7108633/99d36dee-e15a-11e4-8fcc-d93181d2fc13.png)

**Comment by daankortenbach** [comment]
Confirmed, have exactly the same issue.

**Comment by daankortenbach** [comment]
I fixed it by setting this manually and restarting ST:
https://github.com/mediachicken/sublimetext-defaultplus-theme/commit/5cb4d87ad6dbd3d2063f4d6591fca9ac75b5d7d0

**Comment by mediachicken** [comment]
I forgot to mention that it requires a relaunch after it installs.

Let me know if you're still having issues.

**Comment by mediachicken** [comment]
Also have a look at the new theme options in the readme. The old options you mentioned are no longer working.

**Comment by themadcreator** [comment]
same, restart is no help. I even did a full sublimetext re-install and this happens once i install the theme.

**Comment by mediachicken** [comment]
What operating system are you guys using? And is it ST2 or ST3?

**Comment by themadcreator** [comment]
OSX 10.9, SublimeText 2.0.2 Build 2221

**Comment by mediachicken** [comment]
Did you install via git or package control? Also, does this happen for every color theme?

**Comment by themadcreator** [comment]
i tried both, same result. It was initially installed through package control, and i noticed that things went wrong when i opened sublimetext this morning. When package control updated to your latest version, the sidebar color reverted to light gray. Then, the glitches began.

**Comment by daankortenbach** [comment]
This (or another?) theme just updated itself again and showed a notification that graphical glitches would be present until a restart of ST. After restart all ok.

**Comment by abailey-et** [comment]
Yeah I tried all the suggestions nothing works. I'm using the same ST2 Build 2221. I installed via both as well once I started getting errors.

My two cents in I think it's not finding the files somehow. The path is messed up or something. Do this, rename the theme to "foobar". Then restart ST2. You'll get exactly what I'm seeing.

Example:
```javascript
"theme": "foobar"
```

**Comment by mediachicken** [comment]
Hmm, seems I'm getting these same issues in ST2. I tested the beta builds in ST2 and it was working perfectly, I'm investigating now.

**Comment by rmilesson** [comment]
Are there any progress on this? I just updated DefaultPlus and after restart the sidebar is completely b0rkd.
Tested things mentioned above but nothing happens.

Using ST 2.0.2 2221 on a Windows 8.1 machine.

**Comment by rmilesson** [comment]
Tried on my Macbook as well now, same result unfortunately.
